.golangci.yml

```diff
@@ -105,7 +105,7 @@ issues:
       text: "(cyclomatic|cognitive)"
       linters:
         - revive
-    - path: _test\.go
+    - path: _test\.go|^common/persistence\/tests\/.+\.go
```

Comment:
what's this for?

Reply:
Added a comment. Basically, this linter was complaining if I had tests like this:

```go
func TestFoo(t ...) {
  someErr = errors.New("test err") // would complalin that this was dynamically defined and should be static instead
  someMock.EXPECT().Return(someErr)
  _, err = someCode.Call()
  assert.ErrorIs(t, err, someErr)
}
```